src/guides/metamask-api/index.md

```diff
@@ -0,0 +1,1096 @@
+---
+title: "Web3 Unleashed: Build a dapp with Next.js and the MetaMask API"
+hide:
+  - navigation
+---
+# Web3 Unleashed: Build a dapp with Next.js and the MetaMask API
+
+Written by [Eric Bishard](https://twitter.com/httpJunkie)
+
+Last updated 10/12/2022
+
+## Overview
+
+In this edition of [Web3 Unleashed](https://trufflesuite.com/unleashed/), we're interviewing the MetaMask DevRel team [Gui Bibeau](https://twitter.com/guibibeau) and [Eric Bishard](https://twitter.com/httpjunkie) about the MetaMask API and other tools and features on the horizon like their MetaMask SDK. We will be building a dapp live with them to custom integrate MetaMask into a NextJS application!
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/vQVletnhLVk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+If you would like to follow along with the Web3 Unleashed Episode #7 demo during or after the stream, below are the steps we walked through to build our dapp integration with MetaMask API.
+
+## Prerequisites
+
+- [MetaMask Extension](https://metamask.io/download/)
+- NodeJS
+- NPM
+- Git
+
+## Clone and Install Dependencies
+
+Clone the [MetaMask API Web3 Unleashed repo](https://github.com/GuiBibeau/Metamask-Nextjs-Starter). Then, to get your project started, change into the directory and install the dependencies:
```

Comment:
Thoughts on taking a fork of this under the `trufflesuite` or `metamask` org?

Reply:
Yeah we can do that.